src/main/java/api/LabelsController.java

```diff
@@ -44,8 +47,9 @@ public void defaultLabels(HttpServletRequest req, HttpServletResponse resp) thro
         resp.setCharacterEncoding("UTF-8");
 
         String labels = new Gson().toJson(
```

Comment:
you definitely dont need to do this. I'll merge it down but will show you a WAY easier way to return json.